### ISSUE TITLE
MVP component selection: cloud storage/drive app

### ISSUE THREAD

**Issue by ghost** [getsubrosa/subrosa#4]

**Comment by nh-github** [comment]
What is the goal for this - bulk storage, an multi-user interactive workflow, both, or something else?
I have used SyncThing a little and it has been okay, but I am using it for direct host<->host file transfer (not putting things online).  Owncloud caught my attention for the ``put something online to share with others via simple href'' -- is that something we want to facilitate?  I decided against sparkleshare because it depends on ms.net/mono.  
git-annex looks interesting, but depends on Haskell.

**Comment by talexand** [comment]
Yeah, there are two main criteria to satisfy: meeting users and having the narrowest dependencies.

I think sharing things online can be broken out to a separate task: both SpiderOak and Dropbox acknowledge or have been publicly called out for the loss of privacy that comes from their features in that vein. Users should be a little bit conscious of that.

**Comment by nh-github** [comment]
SyncThing and Seafile feel very different to me and their dependencies differ wildly as well.  Seafile needs django and friends for runtime.  SyncThing requires a web browser for runtime configuration.

**Comment by ryankanno** [comment]
I have an owncloud role setup. It's not fully featured as to scale owncloud, you need a few machines. I can setup each machine, but as a first go, I'm going to check this in.

**Comment by talexand** [comment]
@ryankanno ??

**Comment by ryankanno** [comment]
Oh, I'm going to check in an owncloud role so we can start at least looking at some of these things.  Maybe we need another playbook to test things, otherwise I don't get the feeling we're going to move on some of these topics. I can definitely check it in on a branch.
